src/app/core/services/issue-comment.service.ts

```diff
@@ -85,8 +88,13 @@ export class IssueCommentService {
   /**
    * To add/update an issue.
    */
-  updateLocalStore(commentsToUpdate: IssueComments) {
-    this.comments.set(commentsToUpdate.issueId, commentsToUpdate);
+  updateLocalStore(commentToUpdate: IssueComment, issueId: number) {
+    const issueComments = <IssueComments>{
+      issueId: issueId,
+      comments: [],
```

Comment:
Why do we update with an empty array? Will there be a lost of information if there are other comments?

Reply:
This is to reduce the load time for the parsing of issue comment as only one of the comment (with the response) is important